src/app.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+'use strict';
+let fs = require('fs');
+let path = require('path');
+const chalk = require('chalk');
+const fetch = require('node-fetch');
+
+let uri = process.argv[2]
+let lengthProcess = process.argv.length
+let stats = process.argv[3];
+
+
+const controller = (arrLinks) => {
```

Comment:
Ser más especifica con el nombre de la funcion
```suggestion
let controllerProcess = (arrLinks) => {
```